### DISCUSSION TITLE
11 Mini QR Code Bug Labs FW2.30.70

### DISCUSSION THREAD

**Discussion by vegas6456** [gopro/labs#403] [Q&A]
Just got 2x 11 mini to go along with my 11 black for FPV use. When I put Labs on the Minis, the QR codes do not work. I have tried factory reset, reinstall normal firmware, then install Labs, multiple times.

What occurs is I present the camera a QR code, and on the cameras OLED screen it shows the QR code with a checkmark. But, no beeps, and no settings change no matter the QR code.  

It seems like the camera recognizes and reads the QR code but does not execute any changes based on the code presented. 

Has anyone else encountered this?

**Comment by dnewman-gpsw** [comment]
I have not.  What QR codes are you scanning?  Please share.
